workflow/schemas/config.schema.yaml

```diff
@@ -105,15 +105,9 @@ properties:
         items:
             type: string
         description: list of chromosomes that should be skipped, used when determining which chr bam files will be merged
-    required:
```

Comment:
why is the required removed?

Reply:
you can basically not run the RNA part without the DNA part if we kept this

Reply:
yes! But if you run the DNA-pipeline and fails to include everything you will not be warned about missing entries. We should maybe have separate entry for rna reference?

Reply:
Now we have different schemas for dna and rna reference

Reply:
Me and Martin discussed this issue yesterday and couldn't agree on a best solution. 

I have a suggestion of an alternative solution. We go back to having one schema and and move out the bwa specifc part to an new entry, baw_reference (or sometthing similar). Which makes it possible to set  the indexes required if one include bwa_reference, even though bwa_reference isb't reqiured be the schema. And also add a star_reference, with star specific parts.

 I believe it would be best if the modules could validate as much stuff as possible, but we could leave it until we can make it possible for all modules.

Reply:
I have now put the indexes under bwa_mem rule and set them as required

Reply:
I have now put the indexes under bwa_mem rule and set them as required